src/WebJobs.Script/Workers/ProcessManagement/WorkerDescription.cs

```diff
@@ -45,6 +45,11 @@ public abstract class WorkerDescription
         /// </summary>
         public abstract bool UseStdErrorStreamForErrorsOnly { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the worker description is disabled.
+        /// </summary>
+        public bool? IsDisabled { get; set; }
```

Comment:
What is the purpose of nullable here?

Reply:
See the [comment in the original PR](https://github.com/Azure/azure-functions-host/pull/10231#discussion_r1640549477).